Author: Jamie Strandboge <jamie@canonical.com>
Description: allow writes to /{,var/}run/sendsigs.omit.d/*dnsmasq.pid for
 network manager integration
Bug-Ubuntu: https://launchpad.net/bugs/941808

Index: apparmor-2.7.99/profiles/apparmor.d/usr.sbin.dnsmasq
===================================================================
--- apparmor-2.7.99.orig/profiles/apparmor.d/usr.sbin.dnsmasq	2012-03-02 08:13:38.000000000 -0600
+++ apparmor-2.7.99/profiles/apparmor.d/usr.sbin.dnsmasq	2012-03-02 09:53:59.000000000 -0600
@@ -53,6 +53,7 @@
 
   # NetworkManager integration
   /{,var/}run/nm-dns-dnsmasq.conf r,
+  /{,var/}run/sendsigs.omit.d/*dnsmasq.pid w,
 
   # Site-specific additions and overrides. See local/README for details.
   #include <local/usr.sbin.dnsmasq>
